src/openzaak/components/zaken/tests/test_zaken.py

```diff
@@ -943,6 +943,91 @@ def test_betalingsindicatie_weergave(self):
             zaak6["betalingsindicatieWeergave"], BetalingsIndicatie.nvt.label
         )
 
+    def test_zaak_autofill_einddatums(self):
+        url = reverse("zaak-list")
+        startdatum = date(2020, 1, 1)
+        self.zaaktype.servicenorm_behandeling = relativedelta(days=30)
+        self.zaaktype.save()
+
+        response = self.client.post(
+            url,
+            {
+                "zaaktype": f"http://testserver{self.zaaktype_url}",
+                "bronorganisatie": "517439943",
+                "verantwoordelijkeOrganisatie": "517439943",
+                "registratiedatum": str(startdatum),
+                "startdatum": str(startdatum),
+            },
+            **ZAAK_WRITE_KWARGS,
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED, response.data)
+        zaak = Zaak.objects.get(uuid=response.data["uuid"])
+
+        self.assertEqual(
+            zaak.einddatum_gepland,
+            startdatum + relativedelta(days=30),
+        )
+        self.assertEqual(
+            zaak.uiterlijke_einddatum_afdoening,
+            startdatum + relativedelta(days=30),
```

Comment:
In this case both the servicenorm and doorlooptijd are the same, so technically it could be that the same variable is used in both calculations. Could you set a different `servicenorm_behandeling` to make sure that's not the case?